src/construction/Slider/slider.scss

```diff
@@ -0,0 +1,138 @@
+
+.flickity-enabled {
+	position: relative;
+	height: auto;
+ }
+ 
+ .flickity-enabled:focus { 
+	outline: none;
+
+}
+ 
+ .flickity-viewport {
+	overflow: hidden;
+	position: relative;
+	height: auto;
+ }
+ 
+ .flickity-slider {
+	position: absolute;
+	width: 100%;
+	height: 100%;
+ }
+ 
+ /* draggable */
+ 
+ .flickity-enabled.is-draggable {
+	-webkit-tap-highlight-color: transparent;
+	-webkit-user-select: none;
+		-moz-user-select: none;
+		 -ms-user-select: none;
+			  user-select: none;
+ }
+ 
+ .flickity-enabled.is-draggable .flickity-viewport {
+	cursor: move;
+	cursor: -webkit-grab;
+	cursor: grab;
+ }
+ 
+ .flickity-enabled.is-draggable .flickity-viewport.is-pointer-down {
+	cursor: -webkit-grabbing;
+	cursor: grabbing;
+ }
+ 
+ /* ---- flickity-button ---- */
+ 
+ .flickity-button {
+	position: absolute;
+	background: hsla(0, 0%, 100%, 0.75);
+	border: none;
+	color: #333;
+ }
+ 
+ .flickity-button:hover {
+	background: white;
+	cursor: pointer;
+ }
+ 
+ .flickity-button:focus {
+	outline: none;
+	box-shadow: 0 0 0 5px #19F;
+ }
+ 
+ .flickity-button:active {
+	opacity: 0.6;
+ }
+ 
+ .flickity-button:disabled {
+	opacity: 0.3;
+	cursor: auto;
+	/* prevent disabled button from capturing pointer up event. #716 */
+	pointer-events: none;
+ }
```

Comment:
I would remove this comment, a developer that know CSS will know that `pointer-events: none` is